src/liballoc/raw_vec.rs

```diff
@@ -201,14 +201,20 @@ impl<T, A: Alloc> RawVec<T, A> {
     ///
     /// This will always be `usize::MAX` if `T` is zero-sized.
     #[inline(always)]
-    pub fn cap(&self) -> usize {
+    pub fn capacity(&self) -> usize {
         if mem::size_of::<T>() == 0 {
             !0
         } else {
             self.cap
         }
     }
 
+    // For backwards compatibility
+    #[inline(always)]
```

Comment:
Instead of `#[inline(always)]` could this be `#[doc(hidden)]` and `#[rustc_deprecated]`?

Reply:
In case you are asking me: I'm a total Rust newbie, and I don't know what should be used here.

Just tell me, and I can change the PR. Or feel free to make any changes yourself.

@Mark-Simulacrum mentioned the `RawVec` is a private internal type and we don't need to keep compatibility?